common_benchmark_suite/openxla/benchmark/comparative_suite/jax/benchmark_definitions.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import itertools
+
+from openxla.benchmark import def_types, unique_ids
+from openxla.benchmark.devices import gcp_devices
+from openxla.benchmark.comparative_suite import utils
+from openxla.benchmark.comparative_suite.jax import model_definitions, test_data_definitions
+
+T5_LARGE_JAX_A2_HIGHGPU_1G = utils.build_batch_benchmark_cases(
```

Comment:
Should this have FP32 and input size in the name? `T5_LARGE_FP32_JAX_523XI32_A2_HIGHGPU_1G`. It's long but it's possible that we'll have another benchmark with the same model but different input sequence length.